cdap-client-tests/src/test/java/co/cask/cdap/client/ClientTestsSuite.java

```diff
@@ -29,7 +29,9 @@
 @RunWith(Suite.class)
 @Suite.SuiteClasses({ ApplicationClientTestRun.class, DatasetClientTestRun.class, MetricsClientTestRun.class,
                       MonitorClientTestRun.class, ProcedureClientTestRun.class, ProgramClientTestRun.class,
-                      QueryClientTestRun.class, StreamClientTestRun.class, ServiceClientTestRun.class })
+                      QueryClientTestRun.class, StreamClientTestRun.class, ServiceClientTestRun.class,
+                      PingClientTestRun.class, /*AdapterClientTestRun.class, */NamespaceClientTestRun.class,
```

Comment:
remove the comment? The changes in this PR still don't allow AdapterclientTestRun to be a part of the suite.